main.py

```diff
@@ -157,42 +142,112 @@ def add_menu(self):
+                              ["5. Видалити контакт з книги"],
+                              ["6. Повернутись в попереднє меню"]])
+        return show_delete
 
+    @property
+    def notes_menu(self):
+        show_notes_menu = ColorTable(theme=Themes.OCEAN)
+        show_notes_menu.field_names = [f"{18 * '-'}Нотатки{18 * '-'}"]
+        show_notes_menu.hrules = 1
+        show_notes_menu.align = "l"
+        show_notes_menu.add_rows([["1. Подивитись всі нотатки"],
+                                      ["2. Додати нотатку"],
+                                      ["3. Знайти нотатку"],
+                                      ["4. Змінити нотатку"],
+                                      ["5. Видалити нотатку"],
+                                      ["6. Повернутись в попереднє меню"]])
+        return show_notes_menu
+
+
+class Handler:
     def __init__(self):
+        self.menu = Menu()
+        self.notes_book = NotesBook()
+        self.address_book = AddressBook()
 
-        self.Adderessbook = AddressBook()
-        self.notes = []  # a func add_note should create an instance of Note
-        # and add it to current list
-        self.menu = Menu()  # should be changed
+    def action_note(self, notes_book: NotesBook):
+        self.notes_book = notes_book
+        while True:
+            print(self.menu.notes_menu)
+            action = input("\033[34m" + "Обери потрібну команду(1-5), "
+                                         "або я спробую вгадати: ")
+            if action.lower() in ["1", "check", "подивитись", "нотатки",
+                                     "замітки", "заметки"]:
+                print(self.notes_book.data)
+            elif action.lower() in ["2", "create", "створити", "создать",
+                                     "замітки", "заметки"]:
+                note = Note(input('Введіть нонатку: '))
+                self.notes_book.add_note(note)
+            elif action.lower() in ["3", "знайти", "search", "нотатки",
+                                     "замітки", "заметки"]:
+                pass
+                #self.notes_book.add_note(note)
+            elif action.lower() in ["exit", "close", "good bye", "5", "вихід",
+                                    "выход"]:
+                break
+            else:
+                print('You was wrong or notes didn\'t create')
+
+    def action_phone(self, address_book: AddressBook):
+        pass
 
+    def main_action(self):
         while True:
             print(self.menu.main_menu)
-            command = input("I am waiting for your command: ")
 
-            if command.lower() in ["exit", "close", "good bye", "11", "вихід", "выход"]:
+            command = input("\033[34m" + "Обери потрібну команду(1-5), "
+                                         "або я спробую вгадати: ")
+            if command.lower() in ["exit", "close", "good bye", "5", "вихід", "выход"]:
                 print("Good bye!")
+                return 'break'
```

Comment:
it shall return str object "break". please delete return and brackets or simply leave return(the function will stop without returning anything)